system/jlib/jstring.cpp

```diff
@@ -980,8 +978,8 @@ StringBuffer & StringBuffer::replaceString(const char* oldStr, const char* newSt
     if (curLen)
     {
         StringBuffer temp;
-        size32_t oldlen = oldStr ? strlen(oldStr) : 0;
-        size32_t newlen = newStr ? strlen(newStr) : 0;
+        size_t oldlen = oldStr ? (size_t)strlen(oldStr) : 0;
```

Comment:
I don't think these casts should be needed, but recently added.

Reply:
will remove